zeroex/order.go

```diff
@@ -142,7 +142,7 @@ type OrderEvent struct {
 }
 
 type orderEventJSON struct {
-	Timestamp                time.Time            `json:"timestamp"`
+	Timestamp                string               `json:"timestamp"`
```

Comment:
In any Go type this should probably still be `time.Time`. Was there a reason to make this change?

Reply:
Yep, I made this change so that we could unmarshal the json from stringifying an `OrderEvent`. We can't convert `OrderEvent` to a JSValue without changing this to a string, so I also thought that it would make sense for this to be consistent.

Reply:
This type is supposed to help with marshaling to and from JSON, in which case `time.Time` works perfectly fine. It's only converting to and from `JSValue` that causes problems. If we're using this type for `JSValue` conversions, we should rename it.